### DISCUSSION TITLE
How can I distribute FAISS over several computers if it is larger that one's PC RAM.

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [facebookresearch/faiss#2420] [Q&A]
How can I distribute FAISS over several computers if it is larger that one's PC RAM.

**Comment by sstone-codaio** [comment]
You can look at https://github.com/facebookresearch/faiss/issues/1037

**Comment by mausch** [comment]
https://opensearch.org/docs/latest/search-plugins/knn/approximate-knn/
